src/pages/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Button } from "../components/atomic";
+import { Button, Checkbox } from "../components/atomic";
```

Comment:
In the future, create a folder called "test" under pages and create test pages there. This will reduce merge conflicts and make it easier to clean stuff up later. (This is what I've been doing)